plugins/c8y_configuration_plugin/src/main.rs

```diff
@@ -501,7 +516,7 @@ mod tests {
             &mut tedge_command_messages,
             TEST_TIMEOUT_MS,
             &[format!(
-                "/tedge/file-transfer/{child_device_id}/config_snapshot/{config_type}"
+                "/tedge/{child_device_id}/config_snapshot/{config_type}"
```

Comment:
Why was `file-transfer` path removed from here when it's used in the `child_device` module?